app/views/vacancies/_filters.html.slim

```diff
@@ -47,4 +57,5 @@ ruby:
   = render(FiltersComponent.new(form: f,
     filters: { total_count: @jobs_search_form.total_filters, title: "Filter results" },
     items: items,
-    options: { remove_buttons: true, mobile_variant: true, close_all: true }))
+    options: { remove_buttons: true, mobile_variant: true, close_all: true },
+    html_attributes: { tabindex: "-1", id: "filters-component" }))
```

Comment:
Why tab-index -1?

Reply:
it is so that element is focusable (as it is a not normally focusable element) the js sets focus when the side panel is opened